opm/simulators/linalg/setupPropertyTree.cpp

```diff
@@ -241,6 +241,16 @@ setupPropertyTree(FlowLinearSolverParameters p, // Note: copying the parameters
         return setupILU(conf, p);
     }
 
+    // mixed-precision ILU0
+    if (conf == "mixed-ilu0") {
+        return setupMixedILU(conf, p);
+    }
+
+    // mixed-precision ILU0
+    if (conf == "mixed-dilu") {
+        return setupMixedDILU(conf, p);
```

Comment:
The error message below should probably be adapted.

Also we need to skip this or fallback to something different once my PR is merged and we do not have avx2 .